axis.go

```diff
@@ -44,6 +44,9 @@ type Axis struct {
 		// counterclockwise will be added to the label
 		// text before drawing.
 		draw.TextStyle
+
+		// Position is where the axis label string should be drawn.
```

Comment:
I think this needs documentation explaining that it's relative to the center and that -1 and 1 are the ends.